src/pages/[platform]/build-a-backend/data/working-with-files/index.mdx

```diff
@@ -210,13 +254,28 @@ if (!updatedSong?.coverArtPath) return;
 
 // Retrieve the file's signed URL:
 const signedURL = await getUrl({ path: updatedSong.coverArtPath });
-
 ```
 
+</InlineFilter>
+
 ## Query a record and retrieve the associated file
 
 To retrieve the file associated with a record, first query the record, then use Storage to get the signed URL. The signed URL can then be used to download the file, display an image, etc:
+<InlineFilter filters={["swift"]}>
+```swift title="ContentView"
+// Get the song record
+let song = try await Amplify.API.query(request: .get(Song.self, byId: currentSong.id)).get()
```

Comment:
```suggestion
let song = try await Amplify.API.query(request: .get(Song.self, byIdentifier: currentSong.id)).get()
```

`byIdentifier` is the preferred API, the `byId` still works but is deprecated since introducing custom primary keys